projects/metis/src/app/dashboard/filter-ops/filter-ops-conf.ts

```diff
@@ -12,6 +12,125 @@ import { RenameWorkflowPipe } from '../../_translate';
 
 const today = new Date().toISOString().split('T')[0];
 
+/**
+ * setDateFrom
+ *
+ * Function to invoke when the date-from is set
+ *
+ * @param { CanHaveError } - cmp - the component
+ * @param { HTMLInputElement } - el - the native element
+ * @param { HTMLElement[] } - opElements - elements related in the config group
+ **/
+const setDateFrom = (cmp: CanHaveError, el: HTMLInputElement, opElements?: HTMLElement[]): void => {
+  const dates = isDateSupported() && el.getAttribute('type') === 'date';
+  const val = el.value;
+  const max = el.getAttribute('max');
+
+  if (val && max) {
+    if (dates && val > max) {
+      el.value = max;
+      el.dispatchEvent(new Event('change'));
+      return;
+    } else if (!dates) {
+      const valid = isValidDate(val);
+      if (!valid) {
+        cmp.setHasError(true);
+        return;
+      } else {
+        cmp.setHasError(false);
+        if (new Date(val) > new Date(max)) {
+          el.value = max;
+          el.dispatchEvent(new Event('change'));
+          return;
+        }
+      }
+    }
+  } else {
+    cmp.setHasError(false);
+  }
+
+  // Add/remove the min value attribute to/from the related html elements
+  if (opElements) {
+    opElements.forEach((item) => {
+      if (item.id === 'date-to') {
+        if (val) {
+          item.setAttribute('min', val);
+        } else {
+          item.removeAttribute('min');
+        }
+      }
+    });
+  }
+};
+
+/**
+ * setDateTo
+ *
+ * Function to invoke when the date-from is set
+ *
+ * @param { CanHaveError } - cmp - the component
+ * @param { HTMLInputElement } - el - the native element
+ * @param { HTMLElement[] } - opElements - elements related in the config group
```

Comment:
According to my IDE this is not valid JSDoc: the first hyphen needs to be removed in all 3 lines.